README.md

```diff
@@ -163,6 +166,190 @@ Then the page should be accessible checking only: document-title, label
+	{iframe: '#frame2', selector: '#selector2'}
+)
+
+# Simple selectors *and* IFrame selector
+# Include the element with id 'selector1' *and* all elements with class 'selector2'
+# Include the element with id 'selector3' inside the IFrame with id 'frame'
+expect(page).to be_accessible.within '#selector1', '.selector2', iframe: '#frame', selector: '#selector3'
+
+# Nested IFrame selectors
+# Include the element selector1 inside the IFrame with id 'frame2',
+# inside the IFrame with id 'frame1'
+expect(page).to be_accessible.within(iframe: '#frame1', selector: 
+	{iframe: '#frame2', selector: '#selector1'}
+)
+```
+
+The inclusion clause `within '#selector'` specifies which elements of the page should be checked. A valid [CSS selector][css selector] must be provided. Within accepts a single selector, an array of selectors, or a hash describing iframes with selectors.
+
+*see additional [context parameter documentation][context-param]*
+
+### Exclusion clause
+
+``` ruby
+# Simple selector
+expect(page).to be_accessible.excluding '#selector1'
+
+# Compound selector
+# Exclude all elements with the class 'selector2' inside the element with id 'selector1'
+expect(page).to be_accessible.excluding '#selector1 .selector2'
+
+# Multiple selectors
+# Exclude the element with id 'selector1' *and* all elements with class 'selector2'
+expect(page).to be_accessible.excluding '#selector1', '.selector2'
+
+# IFrame selector
+# Exclude the element with id 'selector1' inside the IFrame with id 'frame1'
+expect(page).to be_accessible.excluding iframe: '#frame1', selector: '#selector1'
+
+# Multiple IFrame selectors
+# Exclude the element with id 'selector1' inside the IFrame with id 'frame1'
+# Exclude the element with id 'selector2' inside the IFrame with id 'frame2'
+expect(page).to be_accessible.excluding(
+	{iframe: '#frame1', selector: '#selector1'},
+	{iframe: '#frame2', selector: '#selector2'}
+)
+
+# Simple selectors with IFrame selector
+# Exclude the element with id 'selector1' *and* all elements with class 'selector2'
+# Exclude the element with id 'selector3' inside the IFrame with id 'frame'
+expect(page).to be_accessible.excluding '#selector1', '.selector2', iframe: '#frame', selector: '#selector3'
+
+# Nested IFrame selectors
+# Exclude the element selector1 inside the IFrame with id 'frame2',
+# inside the IFrame with id 'frame1'
+expect(page).to be_accessible.excluding(iframe: '#frame1', selector: 
+	{iframe: '#frame2', selector: '#selector1'}
+)
+```
+
+The exclusion clause `excluding '#selector'` specifies which elements of the document should be ignored. A valid [CSS selector][css selector] must be provided. Excluding accepts a single selector, an array of selectors, or a hash describing iframes with selectors.
+
+*see additional [context parameter documentation][context-param]*
+
+### Accessibility Standard (Tag) clause
+
+```ruby
+# Single standard
+expect(page).to be_accessible.according_to 'wcag2a'
```

Comment:
I would prefer to use symbols for all the tags and rule names unless you have a reason not to?

Though we should probably verify for certain that both are accepted. (I'm pretty sure we `to_s` the tags/rules.)